test/test_utils/eks.py

```diff
@@ -302,26 +308,20 @@ def eks_setup(framework, cluster_name=None):
 
     eks_tools_installed = not run_out.return_code
 
-    # Assume cluster with such a name is active
-    if not cluster_name:
-        eks_cluster_name = PR_EKS_CLUSTER_NAME_TEMPLATE.format(framework)
-    else:
-        eks_cluster_name = cluster_name
-
     if eks_tools_installed:
-        eks_write_kubeconfig(eks_cluster_name, "us-west-2")
         return
 
     platform = run("uname -s").stdout.strip()
 
     kubectl_download_command = (
         f"curl --silent --location https://amazon-eks.s3-us-west-2.amazonaws.com/"
-        f"{EKS_VERSION}/2019-08-14/bin/{platform.lower()}/amd64/kubectl -o /tmp/kubectl"
+        f"{EKS_VERSION}/2019-08-22/bin/{platform.lower()}/amd64/kubectl -o /usr/local/bin/kubectl"
```

Comment:
The change in the date, or the change in the target path?

Reply:
The date was taken from https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html, as pointed out by @TusharKanekiDey .

I changed the target path because it reduces the number of steps in the code.